inst/base/mrgsolve-evtools.h

```diff
@@ -34,6 +34,21 @@ void infuse(databox& self, const double amt, const int cmt, const double rate) {
   return;
 }
 
+mrgsolve::evdata replace(const double amt, const int cmt) {
+  mrgsolve::evdata ev(0, 8); 
```

Comment:
The signature for the constructor is `time` and `evid`.  We use `8` here because that signals to zero out the compartment and add the new amount.